requirements.txt

```diff
@@ -1,10 +1,6 @@
 click >= 6.6
 cloudpickle >= 1.5.0
-contextvars;python_version<'3.7'
-# TODO: Update our dask version requirement to
-# the latest dask release before releasing distributed
-# xref https://github.com/dask/distributed/pull/4489
-dask @ git+https://github.com/dask/dask.git@master
+dask @ git+https://github.com/dask/dask.git@main
```

Comment:
It looks like this is causing a merge conflict; should I just resolve this by going with the latest from main?

Reply:
Yes, pull latest from main